RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py

```diff
@@ -29,4 +29,9 @@
                                                        872423215,872416066,872435036,872439336,
                                                        872420273,872436907,872420147,872439731,
                                                        872436657,872420397,872439732,872439339,
-                                                       872439603,872422436])
+                                                       872439603,872422436,872439861,872437051,
+                                                       872437052,872420649,872422436,872421950,
```

Comment:
872422436 is showing up twice.
Please check that this is just a redundancy and not a typo missing some other detid.

Reply:
@slava77  I just checked and I confirm that this is a redundancy, not a typo. This was the first "new" crystal spotted in 2018. We then converged on a more exhaustive list and for some reasons I copied this crystal twice. Sorry for that.